rsc/src/test/java/com/cloudera/livy/rsc/rpc/TestRpc.java

```diff
@@ -186,6 +188,31 @@ public void testEncryption() throws Exception {
     assertEquals(outbound.message, reply.message);
   }
 
+  @Test
+  public void testPortRange() throws Exception {
+    String portRange = "a~b";
+    emptyConfig.set(LAUNCHER_PORT_RANGE, portRange);
+    try {
+      autoClose(new RpcServer(emptyConfig));
+    } catch (Exception ee) {
+      assertTrue(ee instanceof NumberFormatException);
+    }
+    portRange = "11000";
+    emptyConfig.set(LAUNCHER_PORT_RANGE, portRange);
+    try {
+      autoClose(new RpcServer(emptyConfig));
+    } catch (Exception ee) {
+      assertTrue(ee instanceof ArrayIndexOutOfBoundsException);
+    }
+    portRange = "11000~11001";
```

Comment:
Could you use 11000~11110 ? Because 11000~11001 is likely to fail the unit test if these 2 port are happenly being used in the travis server.